data/scenarios/Testing/795-prerequisite/795-prerequisite-cycle-with-not.yaml

```diff
@@ -0,0 +1,46 @@
+version: 1
+name: |
+  Prerequisite objectives: non-dependency cycle due to the NOT
+description: |
+  This should NOT be rejected by the parser.
+  The two objectives do reference each other,
+  but the NOT is a distinct reference from the non-negated goal.
+objectives:
+  - goal:
+      - Make a "furnace".
+    condition: |
+      as base {has "furnace"};
+    id: have_furnace
+    prerequisite:
+      logic:
+        not:
+          id: have_gear
```

Comment:
This may be another common case. Maybe this would read better:
```yaml
prerequisite:
  not: have_gear
```
@byorgey maybe there is an English word for this concept? :thinking:

Reply:
"showstopper" is the only English word I can think of that conveys the idea of a prerequisite of *not* having something.  But it hardly seems worth adding that as a keyword. :smile: